api/valor_api/backend/metrics/detection.py

```diff
@@ -868,11 +868,19 @@ def compute_detection_metrics(*, db: Session, evaluation_id: int):
         .all()
     )
     model = (
-        db.query(Query(models.Model).filter(prediction_filter).any())  # type: ignore - SQLAlchemy type issue
```

Comment:
is `prediction_filter` guaranteed to only include this one check (`models.Model.name == evaluation.model_name`)?

Reply:
Reverting this